index.html

```diff
@@ -400,6 +448,36 @@
       }
     }
 
+    function calculateRingNumber() {
+      // Minimum ring number after checking all directions is the final ring number.
+      // Initialise to a "high" value for comparison to find minimum.
+      var ringNumber = 5;
+      // currCell is initialised by the toggleCell function.
+      var cell = currCell;
+
+      // Walk in all 6 directions and count the distance to the edge.
+      for (let direction = 0; direction < 6; direction++) {
```

Comment:
haha is it easier to hardcode this? like have a board whose cell value is the ring number

Reply:
Went with hardcoding in the end. Computing at the start needs a 2nd nested loop as the computation cannot be done until the neighbour link up is complete.

Doing this at the start of the game also requires the ring numbers of all 37 cells to be calculated, while doing it at harvest time probably only needs 16-17 calls since we harvest about that many trees in a 4 player game.